frontend/webapp/core/modules.py

```diff
@@ -113,3 +142,27 @@ def get_response_all(self):
 
     def get_response_formatted_all(self):
         return self.get_response_all()
+
+    def update_intended_process_states(self, name, running=False):
+        try:
+            access_token = create_access_token(identity=current_user)
+            graqphql_request_headers = {
+                "Content-Type": "application/json; charset=utf-8",
+                "Authorization": "Bearer {}".format(access_token),
+            }
+            graqphql_request_payload = json.dumps(
+                {
+                    "variables": {"name": name, "running": running},
+                    "operationName": "updateIntendedProcessStates",
+                    "query": intended_process_states_mutation,
+                }
+            )
+            # TODO: use or not verify=False?
+            requests.post(
+                url=GRAPHQL_URI,
+                headers=graqphql_request_headers,
+                data=graqphql_request_payload,
+                verify=False,
```

Comment:
@slowr any ideas on how to handle this? Some deployments use self-signed certificates... Should we add an extra env variable or deal with this in another way?

Reply:
the communication is done internally with `http`. There is no issue there as it's a private network

Reply:
Correct, I realized that afterwards :)